src/common/dataAdaptors/titleAdaptors.js

```diff
@@ -0,0 +1,13 @@
+export function titleForEditing({ item }) {
```

Comment:
I was beginning to think it was superfluous given that the filename and directory include the word adaptor. But if we think it would help with clarity we can stick to that.